src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -556,12 +579,14 @@ case class Dependencies(toMap: Map[MavenGroup, Map[ArtifactOrProject, ProjectRec
           }
         case parts =>
           // This can be split, but may not be:
-          val unsplit = ap.get(a).map(_.lang.unversioned(g, a)).toSet
+          val unsplit = ap.get(a).map(pr => {
```

Comment:
can we rewrite as:
```scala
.map { pr =>

}
```
so, we don't use the inner `{ }` and instead use that on the `.map`

Reply:
Can do!

Since my scala-fu is weak and it couldn't hurt to improve it, is this just a syntactic change (more idiomatic scala), or does it change the meaning? As I look at the code around it, I realize I should have done that from the beginning anyway to match the style of the existing code...